srcpkgs/lsof/patches/fix-epoll-test.patch

```diff
@@ -0,0 +1,85 @@
+This patch has been upstreamed already, and should be
```

Comment:
add a link to it please

Reply:
I think this is it:
https://github.com/void-linux/void-packages/pull/41909/files#diff-122915c48ac80e5e28054e63f87f41571facf87faf46b0700d0d9fa9bb1dc9c7

Reply:
@classabbyamp Here -
https://github.com/lsof-org/lsof/commit/5ca335ff9ae6510cba0a94a62fbaa46709f7cb16

Do I need to mention it in the patch?

Reply:
yes please